packages/nuxt/src/pages/utils.ts

```diff
@@ -249,7 +249,7 @@ export function normalizeRoutes (routes: NuxtPage[], metaImports: Set<string> =
   return {
     imports: metaImports,
     routes: genArrayFromRaw(routes.map((page) => {
-      const file = normalize(page.file)
+      const file = normalize(page.file || '')
```

Comment:
I'm still not sure about this line. Should some random name be generated instead? Or pages without actual file should be filtered out?

Reply:
No, they should not be filtered out here. Nor should a random path be generated. This gets fed back into the source for `#build/routes.mjs`. (If we were to filter out these files then no route would be output without a filename, which I think is the opposite result you would like.)

https://github.com/nuxt/nuxt/blob/6016aef859efe485871fe007d9b572304916ebba/packages/nuxt/src/pages/module.ts#L168-L176

I think we can early return in this case without some of the properties on the object we normally return.

Reply:
Probably fixed that. There are no errors with redirects in the playground as far as I can tell.